apps/client/components/review/MetricList.tsx

```diff
@@ -48,13 +55,16 @@ function MetricList({ targetId, targetName, reviewData, isManagerReport = false
     const comment = event.target.value
 
     const updatedState = [...state]
-    const grade = updatedState.find((gradeData) => gradeData.metric === metricName)
+    const grade = updatedState.find(
+      (gradeData) => gradeData.metric === metricName
+    )
 
     grade.comment = comment
     setState(updatedState)
   }
 
   const handleSaveDraft = async () => {
+    variables.input.isDrafted = true
```

Comment:
Wait, I'm not sure this makes sense. You created a GraphQL input but you didn't update the database model so you're not storing the new variable anywhere.

I actually don't think we need to store this info. It doesn't need to be on the input. The server can just calculate if the review is a draft (by checking if any of the `grades` have `comment !== '' || rating !== 0`), and then GraphQL can just return `isDrafted` without it existing on the model.